### ISSUE TITLE
on first start plugin failed

### ISSUE THREAD

**Issue by igreenfield** [nemwiz/jira-commit-message-intellij-plugin#21]
on first install that line is failing:
```
 val pattern = Pattern.compile(String.format(Locale.US, "%s+-[0-9]+", valueToMatch))
```

**Comment by nemwiz** [comment]
So sorry that the plugin is not working for you.

In order to help me find the issue, can you please provide more info on the following:
- Which version of IntelliJ are you using?
- Which version of the plugin are you using?
- Operating system

P.S. How do you know that this line is causing the issue?

Thanks

**Comment by igreenfield** [comment]
- Which version of IntelliJ are you using?

```
IntelliJ IDEA 2021.1 (Ultimate Edition)
Build #IU-211.6693.111, built on April 6, 2021
Runtime version: 11.0.10+9-b1341.35 amd64
```

- Which version of the plugin are you using?
2.1.1

- Operating system
Windows 10

- How do you know that this line is causing the issue?
 from the stacktrace in idea report issue window

**Comment by nemwiz** [comment]
This issue happens due to a missing configuration. It's been fixed in v2.1.5 and instead of an error, a notification will show up and prompt the user to configure the plugin. I'm closing this.
